public/stylesheets/site/2.0/22-system-messages.css

```diff
@@ -18,6 +18,10 @@ System messages use the following colours:
   float: right;
 }
 
+.message.footnote {
```

Comment:
Sorry it took so long to get back to you on this! I wanted to give this some thought for a couple reasons: first, we like to avoid using chained selectors like `.class.class` to avoid specificity issues; and second, while we don't currently use this combination of classes anywhere else that I can find either, it's so generic I can definitely see us doing so in the future, and in a much different context. 

So I think what I'd suggest is using `.actions + .footnote` as the selector, because it most accurately describes  and targets the problem we're trying to solve: actions and footnotes are both floated, so it's important to clear `.footnote` when its an adjacent sibling of `.actions`. I checked and this is the only place where this situation is currently occurring -- most of our footnotes are adjacent siblings of text areas or text inputs.

When we modify something based on the context it appears in, we try to stick it in a little "context" section below all the basic rules (you can see an example in [08-actions.css](https://github.com/otwcode/otwarchive/blob/master/public/stylesheets/site/2.0/08-actions.css)). This file's divided up a little weirdly, so I'd suggest adding
```
/* contexts */

.actions + .footnote {
 clear: both;
}
```
directly above the big "ADMIN ANNOUNCEMENT BANNERS" section.

Thank you, and my apologies again for the delay!